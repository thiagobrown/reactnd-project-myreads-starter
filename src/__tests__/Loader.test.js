import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import Loader from './../components/Loader'

describe('Testing BookShelfChanger Component', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<Loader />);
  });
  
  it('render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});