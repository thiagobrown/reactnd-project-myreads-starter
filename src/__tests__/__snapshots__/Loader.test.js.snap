exports[`Testing BookShelfChanger Component render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Loading...",
      "className": "loader",
    },
    "ref": null,
    "rendered": "Loading...",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Loading...",
        "className": "loader",
      },
      "ref": null,
      "rendered": "Loading...",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
